background/services/keyring/index.ts

```diff
@@ -331,6 +332,21 @@ export default class KeyringService extends BaseService<Events> {
     return newAddress
   }
 
+  /**
+   * Find keyring associated with an account.
+   *
+   * @param account - the account desired to search the keyring for.
+   */
+  findKeyring = async (account: HexString): Promise<HDKeyring> => {
```

Comment:
Two things here

1. Why are we avoiding normal async method syntax?
2. This should be private using `#` We don't want key material to leave the service outside a very tightly defined API.

Reply:
To be a little more precise here, this should just be:

```suggestion
  async #findKeyring(account: HexString): Promise<HDKeyring> {
```

Reply:
Thanks, I instinctively write arrow functions, changed it to be consistent with other function declarations as suggested.